solr/core/src/java/org/apache/solr/search/grouping/CommandHandler.java

```diff
@@ -245,12 +244,6 @@ public NamedList<NamedList<Object>> processResult(
    */
   private void searchWithTimeLimiter(Query query, ProcessedFilter filter, Collector collector)
       throws IOException {
-    if (queryCommand.getTimeAllowed() > 0) {
```

Comment:
[INFO] [LUCENE-10151](https://issues.apache.org/jira/browse/LUCENE-10151) added timeout support to IndexSearcher via a `IndexSearcher.setTimeout()` method, superseding the TImeLimitingCollector previously offered by Lucene.  TLC was officially deprecated in https://github.com/apache/lucene/pull/13220 and removed for Lucene 10.

That said...

[Q] Is `IndexSearcher.setTimeout()` sufficient for what Solr needs?  AFAIK, we allow users to specify timeouts on a per-query basis, and I can't imagine how an IndexSearcher-wide setting would support that?  Should the "searchWithTimeLimiter" method even exist any more, now that there's nothing here that's time-limiting specific?